blocks/http/CMakeLists.txt

```diff
@@ -0,0 +1,14 @@
+FetchContent_Declare(
+    httplib
+    GIT_REPOSITORY https://github.com/yhirose/cpp-httplib.git
+    GIT_TAG v0.15.1
+)
+FetchContent_MakeAvailable(httplib)
```

Comment:
I'd rather keep this in the top-level CMakeLists.txt so its harder to miss.
Also another dependency we have to keep in sync with opencmw-cpp (but I guess that cannot be avoided...)

Reply:
Agreed and done.
Keeping them in sync should not be too much of a problem, from what I can tell cpp-httplib does not break API often.